demo-dotnet/QuantizationAndStorageOptions/README.md

```diff
@@ -0,0 +1,54 @@
+---
+page_type: sample
+languages:
+  - python
+name: Vector quantization and storage options in C#
+products:
+  - azure
+  - azure-ai-search
+description: |
+  Using Azure.Search.Documents and the Azure SDK for C#, save on storage when using vector quantization
+urlFragment: quantization-storage-c#
+---
+
+# Vector quantization and storage options using C# (Azure AI Search)  
+
+The C# program creates vectorized data on Azure AI Search and demonstrates how to save on storage using the following features
+
+- Quantization
```

Comment:
line 18 (if it's not too wordy) - Built-in scalar quantization that reduces vector index size in memory and on disk.

I like saying "built-in" because it clarifies that AI Search is providing the quantization and it conveys that this task is going to be easy.